content/images/figures/unifying_applications_overview.svg

```diff
@@ -0,0 +1,374 @@
+			M 210 50
+			L 110 50
+			"
+			fill="none"
+			stroke="black"
+			marker-end="url('#arrowhead')"
+		/>
+		<text x="10" y="-2" class="label_font">
+			<textPath href="#drugpath">
+				Interacts
+			</textPath>
+		</text>
+	</g>
+	<g id="drug_side_effects" transform="translate(300,0) scale(0.85)">
+		<image href="http://clipart-library.com/images_k/pill-transparent-background/pill-transparent-background-4.png" x="60" y="10" height="100" width="100" />
+		<circle cx="320" cy="55" r="50" class="node side_effect"/> 
+		<text x="275" y="60" font-size="18" font-weight="bold">
+			Side Effects
+		</text>
+		<path id="side_effects" d="
+			M 160 55
+			L 260 55
+		"
+		fill="none"
+		stroke="black"
+		marker-end="url('#arrowhead')"
+		/>
+		<text x="15" y="-5" class="label_font">
+			<textPath href="#side_effects">
+				Causes
+			</textPath>
+		</text>
+	</g>
+	<g id="drug_repurposing" transform="translate(700,-50) scale(0.52)">
+		<image href="http://clipart-library.com/images_k/pill-transparent-background/pill-transparent-background-4.png" x="10" y="190" height="100" width="100" />
+		<image href="http://clipart-library.com/images/XrinAE4cR.png" x="10" y="10" height="100" width="100" />
+		<circle cx="320" cy="155" r="50" class="node disease" />
+		<text x="290" y="160" class="label_font">
+			Disease
+		</text>
+
+		<path id="treats1" d="
+		M 110 65
+		L 260 125
+		"
+		fill="none"
+		stroke="black"
+		marker-end="url('#arrowhead')"
+		/>
+		<text x="34" y="-2" class="label_font">
+			<textPath href="#treats1">
+				Treats?
+			</textPath>
+		</text>
+
+		<path id="treats2" d="
+		M 110 225
+		L 260 175
+		"
+		fill="none"
+		stroke="black"
+		marker-end="url('#arrowhead')"
+		/>
+		<text x="34" y="-2" class="label_font">
+			<textPath href="#treats2">
+				Treats
+			</textPath>
+		</text>
+	</g>
+</g>
+
+<text x="305" y="300" class="label_font">
+	(b) Pharmaceutical Applications
+</text>
+<text x="355" y="450" class="label_font" text-decoration="underline">
+	Drug-Drug Interactions
+</text>
+<text x="675" y="452" class="label_font" text-decoration="underline">
+	Drug Side Effects
+</text>
+<text x="965" y="450" class="label_font" text-decoration="underline">
+	Drug Repurposing
+</text>
+
+<rect x="300" y="500" width="900" height="200" fill="white" stroke="black" rx="5" ry="3" stroke-dasharray="10"/>/>
+<g id="clinical_applications" transform="translate(350, 550) scale(0.8)">
+	<g id="safe_drug_recommendations" transform="translate(50,0)">
+		<image href="http://clipart-library.com/images_k/pill-transparent-background/pill-transparent-background-4.png" x="0" y="0" height="100" width="100" />
+		<image href="http://clipart-library.com/data_images/12685.png" x="200" y="0" height="100" width="100" />
+		<path id="recommendation" d="
+			M 100 50
+			L 195 50
+		"
+		fill="none"
+		stroke="black"
+		marker-end="url('#arrowhead')"
+		/>
+		<text x="5" y="-2" class="label_font">
+			<textPath href="#recommendation">
+				Perscribe
```

Comment:
```suggestion
				Proscribe
```
I think. Check dictionary to be sure.

Reply:
Turns out correct word is prescribe. [answer here](https://www.vocabulary.com/articles/chooseyourwords/prescribe-proscribe/)
Will manually fix this typo.